sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.Legacy.cs

```diff
@@ -0,0 +1,641 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    ///   This portion of the class defines the transformations needed for interacting
+    ///   with the legacy Text Analytics REST API generated types.
+    /// </summary>
+    internal static partial class Transforms
```

Comment:
ideally, this class should be the same as https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs right?
I mean, I can tell it is not, so can you help me understand the disconnect?

Reply:
The `main` branch mixes type renames and such from the core namespace, sharing the public API with the generated code.   In this form, the legacy API is generated as purely internal and isolated.   Mostly, types moved namespaces and occasionally reverted to the pure generated name.   (renames were mostly things previously hit with the `Internal` suffix)